app/src/main/java/fr/free/nrw/commons/profile/achievements/AchievementsFragment.java

```diff
@@ -510,6 +526,19 @@ private void launchAlert(String title, String message){
                 .create()
                 .show();
     }
+    // Launch Alert with a READ MORE button and clicking it open a custom webpage
```

Comment:
Better to write about the function parameters in Javadoc, helps fellow contributors in understanding its use.